### ISSUE TITLE
avoid sell_signal becoming active on open trade if profit below certain percentage

### ISSUE THREAD

**Issue by klara31** [freqtrade/freqtrade#4664]
## Describe your environment

  * Operating system: freqtrade running in docker on a Linux host (freqtradeorg/freqtrade:stable)
  * Freqtrade Version: 2021.3
  
## Your question
If the current profit of an open trade pair is below a certain threshold, I want my strategy not to sell this pair if the pair hits a sell_signal. I cannot figure out how to deal with this in the strategy, any help would be most appreciated!

**Comment by xmatthias** [comment]
You can do this via a combination of `ask_strategy.sell_profit_only` and `ask_strategy.sell_profit_offset` - which defines the profit after which the sell-signal will become active.

**Comment by klara31** [comment]
Thank you, I missed those parameters. These should make it a lot easier!

How does the offset work, as I can not assign a negative value? 
What I want, is that no sell_signal becomes active if profit is (i.e.) below -5%.

**Comment by xmatthias** [comment]
well it's main design is to only work with a positive offset - as i've not seen any usecase where a sell-signa with -4% is valid, while with -6% it isn't - so most people use this as a "positive" offset only - and that's what it's designed for.

**Comment by klara31** [comment]
Can I put up a change request to make this parameter negative as well?

**Comment by xmatthias** [comment]
sure, i don't see why it shouldn't be allowed to be negative.

**Comment by klara31** [comment]
Do I need to do something for this to happen?

**Comment by xmatthias** [comment]
Well my understanding from the above was that you'd be going to create a PR for this (which apparently is a wrong assumption).

In that case, i'll take care of it - but i don't know when yet (it's not a big work - but it's also not something that i see as urgent either).

**Comment by klara31** [comment]
I'm sorry for not being clear.
 
My coding skills are limited. As far as I can see, in file `freqtrade/constants.py` on line 179, the following has to be changed:
```
'sell_profit_offset': {'type': 'number', 'minimum': -100.0},
```
I have never worked with GitHub, other than using `clone` and `pull`, I doubt if you would like me to make my first change on you repository ;) Thanks for your help!

**Comment by klara31** [comment]
I've created a pull request. As it is the first, please be nice to me... ;)

**Comment by xmatthias** [comment]
Closed by #4683
